framework/resources/Datasources/tpcds/maprdb/json/importTables.sh

```diff
@@ -10,138 +10,138 @@ if [ -n $USERNAME ]
 then
     user=$USERNAME
 else
-    user=`ps -aef | grep Drillbit | grep org.apache.drill | cut -d' ' -f1 | head -1`
+    user=ps -aef | grep Drillbit | grep org.apache.drill | cut -d' ' -f1 | head -1
 fi
 
 hadoop fs -mkdir -p /drill/testdata/tpcds_sf$sf/maprdb/json
 
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/customer
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/customer_address
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/customer_demographics
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/household_demographics
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/item
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/promotion
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/time_dim
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/date_dim
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/store
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/store_sales
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/warehouse
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/ship_mode
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/reason
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/income_band
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/call_center
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/web_site
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/store_returns
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/web_page
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/catalog_page
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/inventory
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/catalog_returns
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/web_returns
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/web_sales
-mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/catalog_sales
+/usr/bin/mapr dbshell drop /drill/testdata/tpcds_sf$sf/maprdb/json/customer
```

Comment:
If you're needing to specify the `/usr/bin` explicitly I'm guessing that the environment is broken, because that directory should be part of `$PATH`

Reply:
The issue is not with script not being found - it was and the script would error out. The MapR script has an issue in my opinion - it does not work well when invoked in some situations.